INN/umbrella-energynewsnetwork
# Search Label changes

## Changes

This pull request makes the following changes:

- removes the `placeholder="Search"` attribute from the search input
- adds a `<label>` for the search label

Desktop:
![Screen Shot 2020-04-24 at 13 41 29](https://user-images.githubusercontent.com/1754187/80242126-d51a9f80-8632-11ea-83aa-a89dc9e04578.png)
Mobile:
![Screen Shot 2020-04-24 at 13 41 38](https://user-images.githubusercontent.com/1754187/80242127-d5b33600-8632-11ea-810a-1524844c83b4.png)


## Why



For #93

## Testing/Questions

Features that this PR affects:

- Search form at https://energynews.test/category/digest/


Questions that need to be answered before merging:

- [ ] Should the shared styles for the two IDs be abstracted into a shared class?

Steps to test this PR:

1. check out the branch
2. visit `/category/digest/`
